examples/gatekeeper/managed-resource/samples/allowed.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: s3.aws.crossplane.io/v1beta1
+kind: Bucket
+metadata:
+  annotations:
+    crossplane.io/composition-resource-name: s3-bucket
+    crossplane.io/external-create-pending: "2022-09-29T16:36:09Z"
+    crossplane.io/external-create-succeeded: "2022-09-29T16:36:10Z"
+    crossplane.io/external-name: standard-object-storage-9mfmk-xgqv8
+  creationTimestamp: "2022-09-29T16:36:08Z"
+  finalizers:
+  - finalizer.managedresource.crossplane.io
+  generateName: standard-object-storage-9mfmk-
+  generation: 2
+  labels:
+    crossplane.io/claim-name: standard-object-storage
+    crossplane.io/claim-namespace: application4
+    crossplane.io/composite: standard-object-storage-9mfmk
+  name: standard-object-storage-9mfmk-xgqv8
+  ownerReferences:
+  - apiVersion: crossplane.adobe.io/v1alpha1
```

Comment:
can we change this to a non-specific reference? ie something that doesnt tie it to adobe?